src/components/runbooks/RunbookRunsPage.js

```diff
@@ -0,0 +1,221 @@
+    const responseBody = {
+      runs: [
+        {
+          id: 434,
+          updated_at: '2016-12-20T16:27:42.150227',
+          created_at: '2016-12-20T16:18:42.150142',
+          status: 'scheduled',
+          runbook: {
+            id: 602,
+            name: 'Demo runbook',
+            regionId: 'east-3.hooli.net',
+            tags: ['Monitoring']
+          }
+        },
+        {
+          id: 435,
+          updated_at: '2016-12-20T16:27:42.150227',
+          created_at: '2016-12-20T16:18:42.150142',
+          status: 'scheduled',
+          runbook: {
+            id: 602,
+            name: 'Demo runbook',
+            regionId: 'east-3.hooli.net',
+            tags: ['Monitoring', 'Databases']
+          }
+        },
+        {
+          id: 436,
+          updated_at: null,
+          created_at: '2016-12-20T16:18:42.150142',
+          status: 'in-progress',
+          runbook: {
+            id: 602,
+            name: 'Demo runbook',
+            regionId: 'east-3.hooli.net',
+            tags: ['Monitoring', 'Databases']
+          }
+        },
+        {
+          id: 437,
+          updated_at: '2016-12-20T16:27:42.150227',
+          created_at: '2016-12-20T16:18:42.150142',
+          status: 'finished',
+          runbook: {
+            id: 602,
+            name: 'Demo runbook',
+            regionId: 'east-3.hooli.net',
+            tags: []
+          }
+        },
+        {
+          id: 438,
+          updated_at: '2016-12-20T16:27:42.150227',
+          created_at: '2016-12-20T16:18:42.150142',
+          status: 'scheduled',
+          runbook: {
+            id: 602,
+            name: 'Demo runbook',
+            regionId: 'east-3.hooli.net',
+            tags: ['Databases']
+          }
+        }
+      ]
+    };
+    runbookRuns.items = responseBody.runs.map((runbookRun) => new RunbookRun(runbookRun));
+  }
+
+  fetchData() {
+    return this.constructor.fetchData(this.props);
+  }
+
+  @observable filters = [
+    {
+      name: 'tag',
+      title: 'Any tag',
+      match: (runbookRun, value) => includes(runbookRun.runbook.tags, value)
+    },
+    {
+      name: 'status',
+      title: 'Any status',
+      match: (runbookRun, value) => runbookRun.status === value
+    },
+    {
+      name: 'search',
+      title: 'Search',
+      match: (runbookRun, value) => runbookRun.runbook.name.indexOf(value) >= 0
+    }
+  ];
+
+  @observable filterValues = {
+    tag: '',
+    status: '',
+    search: ''
+  };
+
+  get filterOptions() {
+    return {
+      tag: this.props.runbookRuns.runbookTags.map((tag) => ({value: tag, title: tag})),
+      status: map(RUNBOOK_RUN_STATUSES, (title, status) => ({value: status, title}))
+    };
+  }
+
+  changeFilter = (name, value) => {
+    this.filterValues[name] = value;
+  }
+
+  render() {
+    const {params: {regionName}} = this.props;
+    const runbookRuns = this.props.runbookRuns.items;
+    const filteredRunbookRuns = runbookRuns.filter(
```

Comment:
This could probably be a computed property to avoid recalculating on every render